azurelinuxagent/common/protocol/restapi.py

```diff
@@ -82,13 +82,6 @@ def __init__(self):
         self.vmAgentManifests = DataContractList(VMAgentManifest)
 
 
-class RequiredFeature(DataContract):
```

Comment:
vmSettings does not expose a Value. Since it is not used anyway, I simplified the code to use simple strings instead of instances of  RequiredFeature